tests/trace/dsl/conftest.py

```diff
@@ -147,6 +146,9 @@ def _insert_project_abc(session: Session) -> None:
             start_time=datetime.fromisoformat("2021-01-01T00:00:00.000+00:00"),
             end_time=datetime.fromisoformat("2021-01-01T00:00:05.000+00:00"),
             attributes={
+                "input": {
+                    "value": "XY%*Z",
```

Comment:
Because `%` and `*` are special symbols in alternative implementations such as `LIKE` and `GLOB`, the test will fail (I tried it) if it's not implemented correctly.